docs/shared/core-features/automate-updating-dependencies.md

```diff
@@ -41,24 +45,22 @@ At this stage, after inspecting the `package.json`, you may wish to manually run
 
 ### Step 2: Running migrations
 
-The next step in the process involves using the `migrate` CLI in order to apply the migrations that were generated in `migrations.json` in the previous step.
-
-Each Nx plugin is able to provide a set of migrations which are relevant to particular versions of the package, and so `migrations.json` will only contain migrations which are appropriate for the update you are currently applying.
-
-The common case is that you will simply apply all migrations from the generated JSON file, exactly as they were generated in the previous step, by running:
+You can now run the actual code migrations that were generated in the `migrations.json` in the previous step.
```

Comment:
It's common to call code modification `codemods`, not migrations.

```suggestion
You can now run the actual codemods that were generated in the `migrations.json` in the previous step.
```

Reply:
yeah I know..but ppl might associate them to actual "codemods" (e.g. the Facebook lib). But I might add it in parentheses so ppl understand.